src/store/model/alternate.ts

```diff
@@ -2,10 +2,6 @@ import {Store} from './store';
 
 export const Alternate: Store = {
 	labels: {
-		inStock: {
```

Comment:
How come you removed this label? If it's the incorrect label, we should update it. Otherwise, the reason we have these is so that we don't get as many false positives.

Reply:
as i mentioned in the testing section of this pr's comment, i am happier with the outcome when only checking for `outOfStock`.
german web store barely get any stock atm compared to the na/ca market and if they get any, their status is more like a "available in x to y days" which makes it impossible to search for in the `inStock` label as we don't have regex search yet and most of the shops don't change their "Add to cart" button with different status(es?); e.g. notebooksbilliger (de) always have a "Jetzt vorbestellen" button which translates to "preorder now".

maybe i don't get the point of your argument here, but for my use case this implementation worked like a charm and i got notifications if some stores expected stock and accepted preorders.

Reply:
So what happens here sometimes is that they change their out of stock labels and we were getting a lot of false positives at first. We then inverted the logic to only search for add to cart, but that also had false positives. Combining the two ensured that we got a proper alert.

> which makes it impossible to search for in the inStock label as we don't have regex search yet

Usually can be found using the `test:brand`. They usually don't change their classes based on an item.

Ref: https://github.com/jef/nvidia-snatcher/pull/141

Reply:
i see, updated the `inStock` label to be more accurate